pkg/util/membercluster_client.go

```diff
@@ -201,6 +201,10 @@ func BuildClusterConfig(clusterName string,
 			return nil, err
 		}
 		clusterConfig.Proxy = http.ProxyURL(proxy)
+
+		if len(cluster.Spec.ProxyHeader) != 0 {
+			clusterConfig.WrapTransport = NewProxyHeaderRoundTripperWrapperConstructor(clusterConfig.WrapTransport, cluster.Spec.ProxyHeader)
```

Comment:
Just a question. there is [a comment](https://github.com/kubernetes/kubernetes/blame/16534deedf1e3f7301b20041fafe15ff7f178904/staging/src/k8s.io/client-go/rest/config.go#L110-L111) on the `WrapTransport`:
> A future release will change this field to an array. Use config.Wrap() instead of setting this value directly.

What do you think of it?  
Given the comment was made 5 years ago, I don't know if this change is still on the plan list of Kubernetes. So, use `config.Wrap()` or not, both are OK to me.

Reply:
I think the latter is better.